### PR TITLE
Russian stringtables finished

### PR DESCRIPTION
**Pull request by EpMAK**
Filled all the missing lines, little errors fixed.

### CODE REVIEW

File: addons/respawn/stringtable.xml [hunk 1]
```diff
@@ -45,31 +45,37 @@
...
             <English>Rallypoint East (Base)</English>
             <German>Sammelpunkt Ost (Basis)</German>
             <Spanish>Punto de reunión Este (Base)</Spanish>
+            <Russian>Точка сбора Красных (База)</Russian>
         </Key>
         <Key ID="STR_ACE_Respawn_RallypointIndependentBase">
             <English>Rallypoint Independent (Base)</English>
             <German>Sammelpunkt Widerstand (Basis)</German>
             <Spanish>Punto de reunión Independiente (Base)</Spanish>
+            <Russian>Точка сбора Независимых (База</Russian>
```
Comment (Tourorist) [inline addons/respawn/stringtable.xml]: The closing bracket is missing.

Comment (EpMAK) [inline addons/respawn/stringtable.xml]: Thanks, the string didn't fit in the table cell.

File: addons/vehiclelock/stringtable.xml [hunk 2]
```diff
@@ -112,4 +112,4 @@
             <Russian>Ключ для открытия большинства машин Гражданских.</Russian>
```
Comment (Tourorist) [inline addons/vehiclelock/stringtable.xml]: @EpMAK, if elsewhere (see respawn strings) Independent side is translated as "Независимых", then maybe it should be here as well?

Comment (EpMAK) [inline addons/vehiclelock/stringtable.xml]: Гражданский is Civilian, rather then Незавизимый means Independent.

Comment (Tourorist) [inline addons/vehiclelock/stringtable.xml]: My bad, was previewing the diff in collapsed view and all I saw was the English INDEP. You are right, it's CIV here.

**Pull request merged**